packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -2285,6 +2285,63 @@ void main() {
       expect(opened, isEmpty);
       expect(closed, isNotEmpty);
     });
+
+    // Regression test for
+    // https://github.com/flutter/flutter/issues/119532#issuecomment-2274705565.
+    testWidgets('Shortcuts of MenuAnchor do not rely on WidgetsApp.shortcuts', (WidgetTester tester) async {
+      // MenuAnchor used to rely on WidgetsApp.shortcuts for menu navigation,
+      // which is a problem for Web because the Web uses a special set of
+      // default shortcuts that define arrow keys as scrolling instead of
+      // traversing, and therefore arrow keys won't enter submenus when the
+      // focus is on MenuAnchor.
+      //
+      // This test verifies that `MenuAnchor`'s shortcuts continues to work even
+      // when `WidgetsApp.shortcuts` contains almost nothing.
+
+      int? pressedItem;
+      await tester.pumpWidget(
+        MaterialApp(
+          // A minimal set of shortcuts that only contains 'enter' that is
```

Comment:
"A minimal set of shortcuts that only contains 'enter' that is needed to verify which button is focused" has two clauses, so I _think_ it needs punctuation or a conjunction. I would consider saying something like "The default MaterialApp shortcuts are cleared to isolate the behavior of the DropdownMenu. The Enter Key emits an ActivateIntent() to provide a method of determining which menu item has focus.